packages/angular_devkit/build_angular/src/browser/schema.json

```diff
@@ -82,13 +82,49 @@
       "default": false
     },
     "sourceMap": {
-      "type": "boolean",
+      "type": ["boolean", "object", "string"],
```

Comment:
to reduce the surface area somewhat, what do you think about removing the string option?  If someone wants hidden then they need to use the object form.  It is an advanced option.

Reply:
I agree.